avm/ptn/azd/aks/main.bicep

```diff
@@ -124,6 +122,54 @@ param containerRegistryRoleName string?
 @description('Optional. The name (as GUID) of the role assignment. If not provided, a GUID will be generated.')
 param aksClusterRoleAssignmentName string?
 
+@description('Optional. Custom configuration of system node pool.')
+param systemPoolConfig object = {}
+
+@description('Optional. Custom configuration of user node pool.')
+param agentPoolConfig object = {}
+
+@description('Optional. Specifies whether the KeyvaultSecretsProvider add-on is enabled or not.')
+param enableKeyvaultSecretsProvider bool = true
+
+@description('Optional. If set to true, getting static credentials will be disabled for this cluster. This must only be used on Managed Clusters that are AAD enabled.')
+param disableLocalAccounts bool = false
```

Comment:
WAF-wise I'd probably recommend to set this to true by default - but as it's a pattern module it's up to you. Just wanted to call it out. Feel free to resolve if you disagree

Reply:
Updated based on your comment.